daemon/storage/windows/windows.go

```diff
@@ -20,27 +20,27 @@ import (
 	"time"
 	"unsafe"
 
-	"github.com/Microsoft/go-winio"
+	winio "github.com/Microsoft/go-winio"
```

Comment:
Looking to see anything obvious causing TP5 to fail. This isn't it (I don't think), but why aliased?